### PR TITLE
Added elu emit function to keras2_emitter and caffe_emitter

### PR DESCRIPTION
**Pull request by BlaiseRitchie**
Added elu emit function to keras2_emitter to fix "KerasEmitter has not supported operator [Elu]." error

### CODE REVIEW

**Comment by rainLiuplus** [comment]
Hi @BlaiseRitchie , thanks for your contribution! Have you tested the accuracy of the conversion results?  We have implemented some testing tools and could you please add some scripts to test your code? You can do it by the following steps:
1. Add test model's config into corresponding parser's architecture_map like [this](https://github.com/Microsoft/MMdnn/blob/master/mmdnn/conversion/examples/tensorflow/extractor.py#L29).

2. Edit [test_table](https://github.com/Microsoft/MMdnn/blob/master/tests/test_conversion_imagenet.py#L792). Add the model name and emitter into corresponding item like this: 
`your_model_name'  : [CaffeEmit, KerasEmit]`.

3. Run the corresponding test_xxx.py like [test_tensorflow.py](https://github.com/Microsoft/MMdnn/blob/master/tests/test_tensorflow.py). Finally it will output test result as: 
```
  error: 8.642673492431641e-07 
  L1 error: 1.3829053e-06
  SNR: 34.28389626117947
  PSNR: 63.29787522393757
```

File: tests/test_caffe.py [hunk 1]
```diff
@@ -7,10 +7,6 @@
 from conversion_imagenet import TestModels
 
 def get_test_table():
-    TRAVIS_CI = os.environ.get('TRAVIS')
```
Comment (linmajia) [inline tests/test_caffe.py]: @BlaiseRitchie , please don't delete these lines of code since they control different model sets to be tested on Travis or your local environment. You could simply add the elutestnet in the following and tests/conversion_imagenet.py (around line 853)

File: tests/test_keras.py [hunk 2]
```diff
@@ -7,9 +7,6 @@
 from conversion_imagenet import TestModels
 
 def get_test_table():
-    if six.PY3:
```
Comment (linmajia) [inline tests/test_keras.py]: @BlaiseRitchie , please don't delete these lines of code since they control different model sets to be tested on Travis or your local environment. You could simply add the elutestnet in the following and tests/conversion_imagenet.py (around line 830)
